lansa-vmss-windows-autoscale-sql-database/mainTemplate.json

```diff
@@ -353,7 +353,7 @@
       },
       "dbMinCapacity": {
          "type": "string",
-         "defaultValue": "0.5",
+         "defaultValue": "",
          "metadata": {
             "description": "Minimal capacity that database will always have allocated, if not paused"
```

Comment:
Minimal capacity that database  => The minimum capacity that the database

Reply:
Text updated